pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -531,7 +531,7 @@ func NewVirtualMachineFlavorCrd() (*extv1.CustomResourceDefinition, error) {
 		Names: extv1.CustomResourceDefinitionNames{
 			Plural:     flavor.PluralResourceName,
 			Singular:   flavor.SingularResourceName,
-			ShortNames: []string{"vmflavor", "vmflavors"},
```

Comment:
IMHO this is changing the API. I would just add the new shorter names. WDYT?

Reply:
I'd agree if the API itself wasn't v1alpha1 at the moment, I also couldn't find any examples of automation within the kubevirt namespace using these. At this point I think we are still okay renaming shortnames.

FWIW this is just some earlier refactoring ahead of a much larger series based on my design proposal for flavors and preferences [1] that's going to introduce many more breaking changes to this API.

[1] https://github.com/kubevirt/community/pull/165

Reply:
Actually that's fine but I'd still like to remove them before this API goes hits beta.

Reply:
Right. By automation, I thought any automation outside of org. This reminds me that we should merge  https://github.com/kubevirt/community/pull/160